README.md

```diff
@@ -231,7 +232,20 @@ interface Iterator {
   }
 }
 
-(paramName: Array | TypedArray | String | Map | Set | IterableObject) => Void
+(paramName: IterableObject) => Void
+```
+
+#### The `TypedArray` Type
+
+It covers these contructors: `Int8Array`, `Uint8Array`, `Uint8ClampedArray`, `Int16Array`, `Uint16Array`, `Int32Array`, `Uint32Array`, `Float32Array`, `Float64Array`.
+
+```TS
+interface TypedArray {
+  new (length: Number) => TypedArrayInstance: Object,
+  new (TypedArrayInstance: Object) => TypedArrayInstance: Object,
+  new (buffer: ArrayBuffer, byteOffset?: Number, length?: Number) => TypedArrayInstance: Object,
+  this.from
```

Comment:
To avoid having to define what's an array-like object—which cannot be realistically done—I am using the existing static function.